src/client/app/Sidebar/UserInfo.js

```diff
@@ -52,6 +64,14 @@ class UserInfo extends React.Component {
       }
     }
 
+    if (user.name && !this.state.rc_percentage) {
+      dSteem.rc.getRCMana(user.name).then(res => {
```

Comment:
do we have enough data to calculate percentage without request?
see last method _calculateManabar (https://github.com/jnordberg/dsteem/blob/master/src/helpers/rc.ts#L42)